integration/integrate-with-slack.adoc

```diff
@@ -0,0 +1,24 @@
+[id="integrate-with-slack"]
+= Integrating with Slack
+include::modules/common-attributes.adoc[]
+:context: integrate-with-slack
+
+toc::[]
+
+[role="_abstract"]
+If you are using Slack, you can forward alerts from {product-title} to Slack.
+
+Following is the high-level integration flow for integrating {product-title} with Slack:
```

Comment:
Here, "following" is a noun that needs to be preceded by "the".  (As it is, "following" would have to mean "after" but that's not accurate.) So maybe something like, "The following steps represent a high-level workflow for integrating {product-title} with Slack:"